src/title/TitleStream.js

```diff
@@ -56,7 +56,8 @@ const TitleStream = (props: Props) => {
       flex: 1,
       flexDirection: 'column',
       alignItems: 'flex-start',
-      justifyContent: 'flex-start',
+      justifyContent: 'center',
```

Comment:
`justifyContent: 'center'` makes sense, and I think `height: '100%'` would probably be fine.

In another direction for the latter, though, I found another option I'm now wondering about: to have `alignItems: 'stretch'` instead of `'center'` on the `View` that wraps `<NavBarBackButton>`, `<Title>`, etc., in `ChatNavBar`. I think it may make more sense for the nav bar to say it wants all its children to stretch from top to bottom, than to say it wants everything centered.

I did come at this from the angle of generally preferring to reach for flexbox in React Native (there are a few [differences](https://reactnative.dev/docs/flexbox) from how it works on the web), and possibly that's not the best thing to reach for here—or I may have landed on the wrong layout choice just trying to do something with flexbox at all. 🙂 But I thought I'd mention it since it was something that worked in my testing.

Reply:
I made this change, although I'm not 100% convinced that it's better. I don't have a strong preference, so I just went ahead and did it.

Reply:
Sure, sounds good; we can always change later.